test/unit/features/options/errorCaptured.spec.js

```diff
@@ -247,4 +247,148 @@ describe('Options errorCaptured', () => {
       expect(store.errors[0]).toEqual(new Error('render error'))
     }).then(done)
   })
+
+  it('should capture error from watcher', done => {
+    const spy = jasmine.createSpy()
+
+    let child
+    let err
+    const Child = {
+      data () {
+        return {
+          foo: null
+        }
+      },
+      watch: {
+        foo () {
+          err = new Error('userWatcherCallback error')
+          throw err
+        }
+      },
+      created () {
+        child = this
+        this.foo = 'bar'
+      },
+      render () {}
+    }
+
+    new Vue({
+      errorCaptured: spy,
+      render: h => h(Child),
+    }).$mount()
+
+    waitForUpdate(() => {
+      expect(spy).toHaveBeenCalledWith(err, child, 'callback for watcher "foo"')
+      expect(globalSpy).toHaveBeenCalledWith(err, child, 'callback for watcher "foo"')
+    }).then(Vue.nextTick(done))
```

Comment:
Shouldn't this be something like

```suggestion
    }).then(() => Vue.nextTick(done))
```

or `then(done)`

Right now the code is equivalent to

```js
    waitForUpdate(() => {
	      expect(spy).toHaveBeenCalledWith(err, child, 'callback for watcher "foo"')
	      expect(globalSpy).toHaveBeenCalledWith(err, child, 'callback for watcher "foo"')
	    })
     Vue.nextTick(done)
```

Reply:
Oops, sorry about that.

Reply:
I fixed it.